### ISSUE TITLE
MotionBuilder Python Support & Documentation

### ISSUE THREAD

**Issue by djhauck** [ue4plugins/MobuLiveLink#20]
It would be extremely helpful to have Python SKD support and documentation for the UE - LiveLink device.

**Comment by klateralus** [comment]
I agree on this one, would be really great to have a python wrapper for that plugin, so it would be possible to set 'Subject' from python or rename the 'Subject Name'

**Comment by KirillFromTechAnimTeam** [comment]
Okay, I've tried to make a python wrapper for the LiveLink, but can't figure out how to properly setup the MakeFile.
Somehow LiveLink uses setup that differs from the setup that MotionBuilder Documentation recommends.
@mattyjams could you please help me with MakeFile setup? :)
